src/main/java/core/basesyntax/Lottery.java

```diff
@@ -0,0 +1,13 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class Lottery extends Ball {
+    public String getRandomBall() {
+        int number;
+        ColorSupplier colorSupplier = new ColorSupplier();
+        Random randomBall = new Random();
+        number = randomBall.nextInt(100);
+        return number + " " + colorSupplier.getRandomColor();
```

Comment:
- 100 -> create constant for this number https://mate-academy.github.io/jv-program-common-mistakes/java-core/oop/lottery
- randomBall -> random
- return Ball instead of String from method 
```suggestion
        ColorSupplier colorSupplier = new ColorSupplier();
        Random random = new Random();
        int number = random.nextInt(MAX_NUMBER);
        return new Ball(colorSupplier.getRandomColor(), number);
```